library/src/trinity/src/sass/tool.scss

```diff
@@ -89,7 +89,13 @@ $success: #7952b3;
 @import "icon-list";
 
 body {
-  padding: 1.5rem;
+  //padding: 1.5rem;
+}
+
+.navIntraTool {
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
```

Comment:
This is failing the build because linting doesn't like the class name convention, I believe.

Reply:
I put one option to globally disable the rule above, but you can also do this

```
// stylelint-disable selector-class-pattern
.navIntraTool {
  display: flex;
  flex-wrap: wrap;
  list-style: none;
}
// stylelint-enable
```